sosww/components/dynamo_db.py

```diff
@@ -316,6 +317,72 @@ def _build_scan_iterator(self, attrs=None, table_name=None, strict=True):
         return response_iterator
 
 
+    def batch_get_items_one_table(self, keys_list, table_name=None, max_retries=0, retry_wait_base_time=0.2):
+        """
+        Gets a batch of items from a single dynamo table.
+        Only accepts keys, can't query by other columns.
+
+        :param list keys_list: A list of the keys of the items we want to get.
+                               Ex: [{'hash_col': '1, 'range_col': 2}, {'hash_col': 3, 'range_col': 4}, ...]
```

Comment:
Why is this a list?
Please explain the behavior and logic better in documentation (and comments in code).

Reply:
What would you use instead of a list?